app/api/srch/shared_params.rb

```diff
@@ -0,0 +1,18 @@
+require 'grape'
+
+module Srch
+  module SharedParams
+    extend Grape::API::Helpers
+
+    params :common do
+      requires :srchString, type: String, documentation: { example: 'Spec' }
```

Comment:
This is fantastic. Thanks!!!

Reply:
@jywarren we added the test back. We'll probably work more on the tests in the next weeks, but just to be more careful, we think it's best to not remove it for now. Good catch!

we've already done the same strategy here in the Typeahead, we are planning to open a PR for that after this is approved!

Thanks! :100: 
ps: can't see why the identation is wrong (codeclimate)